frontend/models/store.ts

```diff
@@ -9,7 +9,7 @@ import {
    MenuItemType,
    ProductListLinkMenuItem,
    SubmenuMenuItem,
-} from './menu';
+} from '@ifixit/ui';
```

Comment:
I think these are all types now... `import type ...`

Reply:
Stated above, MenuItemType is not a type. It is an enum. So, the compiler complains about importing this as type. Should I break out the types and import those separately from the same path?

Reply:
Yes, I'd try splitting it up into two statements (types and non types).

Reply:
Split imports in [53ba854](https://github.com/iFixit/react-commerce/pull/1342/commits/53ba8541d62db5b4ea2f9e0f415f7bc2167169e8)